sessions/introduction.qmd

```diff
@@ -88,71 +95,72 @@ Why do we structure it this way?
     `data-raw/` folder, R scripts/functions in the `R/` folder, and
     documents like R Markdown / Quarto files in `doc/`.
 
-This also supports our workflow and processes, which will be something
-like:
+This structure also supports our workflow and processes throughout the
+course, which will be to:
 
--   Install packages in a project-specific environment to track package
-    dependencies.
--   Follow "function-oriented" workflows, where we use R Markdown /
+-   Track package dependencies in the `DESCRIPTION` file.
```

Comment:
Not sure this is the correct "replacement" of the outdated line mentioning a "project-specific environment"